src/navigation/navigation-events.ts

```diff
@@ -46,6 +54,7 @@ export function navigateToUrl(obj) {
     window.location.hash = destination.hash;
   } else if (current.host !== destination.host && destination.host) {
     if (process.env.BABEL_ENV === "test") {
+      // @ts-ignore
```

Comment:
I kind of like using `@ts-expect-error [text here]` so that if the implementation ever changes and it starts working again, TS will be helpful and tell you to remove this comment. Not a big deal though, just a personal preference.

Reply:
👍 updated